build-tools/build-image/cross/Dockerfile

```diff
@@ -51,14 +53,13 @@ RUN mkdir -p /usr/local/src/protobuf \
 
 # Use dynamic cgo linking for architectures other than amd64 for the server platforms
 # More info here: https://wiki.debian.org/CrossToolchains
-RUN echo "deb http://emdebian.org/tools/debian/ jessie main" > /etc/apt/sources.list.d/cgocrosscompiling.list \
-  && curl -s http://emdebian.org/tools/debian/emdebian-toolchain-archive.key | apt-key add - \
-  && for platform in ${KUBE_DYNAMIC_CROSSPLATFORMS}; do dpkg --add-architecture ${platform}; done \
+RUN echo "deb http://cz.archive.ubuntu.com/ubuntu xenial main universe" > /etc/apt/sources.list.d/cgocrosscompiling.list \
```

Comment:
I think this is a pretty big question of trust. Do we trust this repository?
Is it possible to separate the others to use emdebian and only make s390x use something other?